modules/search-service-impl/src/main/java/org/opencastproject/search/impl/persistence/SearchServiceDatabaseImpl.java

```diff
@@ -477,6 +479,18 @@ public Date getDeletionDate(String mediaPackageId) throws NotFoundException, Sea
     }
   }
 
+  public boolean exist(String mediaPackageId) throws SearchServiceDatabaseException {
```

Comment:
I added this method to be able to check if a mediapackage exists. The "getDeletionDate()" method does another permission check and cannot be used for this purpose.